libfwupdplugin/fu-hid-device.c

```diff
@@ -570,7 +570,7 @@ fu_hid_device_get_report_internal(FuHidDevice *self, FuHidDeviceRetryHelper *hel
 		title = g_strdup_printf("HID::GetReport [wValue=0x%04x, wIndex=%u]",
 					wvalue,
 					priv->interface);
-		fu_dump_raw(G_LOG_DOMAIN, title, helper->buf, helper->bufsz);
+		//		fu_dump_raw(G_LOG_DOMAIN, title, helper->buf, helper->bufsz);
```

Comment:
??

Reply:
Ignore this bit; this just reduces the log spam while debugging.

Reply:
Removed from the PR.